python/pyiceberg/table/sorting.py

```diff
@@ -115,10 +115,12 @@ class SortOrder(IcebergBaseModel):
     The order of the sort fields within the list defines the order in which the sort is applied to the data.
 
     Args:
+      fields (List[SortField]): The fields how the table is sorted.
+
+    Keyword Args:
       order_id (int): An unique id of the sort-order of a table.
-      fields (List[SortField]): The fields how the table is sorted
     """
-
+    
```

Comment:
Can you remove this newline?

Reply:
per request, has bee removed

Reply:
Looks like the linter doesn't agree. Can you add the newline, and remove the spaces. I misread the change.

Reply:
... done...